### PR TITLE
Headings to sentence case for Google style

### PR DESCRIPTION
**Pull request by Ordinant**
plus added .vscode to .gitignore.
This is the first set of doc changes, for 
headings only.

### CODE REVIEW

**Comment by Ordinant** [comment]
Hi @anusudarsan, 
Digging into the build errors, they turn out to be apt-get errors to download some Docker components, not related to my doc changes. Those are ready to review. 

Thanks!

**Review by anusudarsan — APPROVED** [review]

File: docs/index.rst [hunk 1]
```diff
@@ -7,13 +7,13 @@ Presto-Admin
 
 Introduction
 ------------
-Presto-Admin is a tool for installing and managing the Presto query engine on a
-cluster. It provides easy-to-use commands to:
+Presto-Admin is a tool for installing and managing the Presto query engine
```
Comment (mosabua) [inline docs/index.rst:10]: Dash or no dash? see title?

Comment (Ordinant) [reply docs/index.rst:10]: I standardized on init cap and no hyphen for the product name: Presto Admin

and lowercase with hyphen for the command name: presto-admin

I must have missed that one instance.

Comment (Ordinant) [reply docs/index.rst]: Fixed with a single-character push.

Comment (mosabua) [reply docs/index.rst]: Excellent!

**Comment by mosabua** [comment]
Can we get this merged @anusudarsan ?

**Comment by anusudarsan** [comment]
done thanks

**Pull request merged**